docs/section23/Rule23-15.md

```diff
@@ -0,0 +1,32 @@
+
+# Airside System - Rule 23-15 
+
+**Schema Version:** 0.0.34  
+**Mandatory Rule:** True  
+**Rule ID:** 23-15  
+**Rule Description:** Dehumidification reheat shall be the same as the system heating type.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 23 Air-side  
+**90.1 Section Reference:** G3.1.3.18 Dehumidification (Systems 3 through 8 and 11, 12, and 13)  
+**Data Lookup:** None  
+**Evaluation Context:** Building  
+
+**Applicability Checks:**  
+
+1. P-RMR is modeled with at least one air-side system that includes humidistatic controls.  
+
+**Function Calls:**  
+
+
+**Applicability Checks:**  
+- For each hvac_p in the P_RMR: `for hvac_p in P_RMR...HeatingVentilatingAirConditioningSystem:`
+    - get the cooling system: `cooling_system_p = hvac_p.cooling_system`
+    - check to see if the cooling system has a humidification type other than NONE, if so, there are humidification controls and the rule is applicable: `if cooling_system_p.humidification_type != NONE:`
+        - the rule is applicable: `APPLICABLE`
```

Comment:
Should be "undetermined". Also see comment to 23-15

Reply:
updated